pkg/azurefile/nodeserver.go

```diff
@@ -59,6 +59,19 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 	mountPermissions := d.mountPermissions
 	context := req.GetVolumeContext()
 	if context != nil {
+		// token request
+		if context[serviceAccountTokenField] != "" && hasClientID(context) {
+			klog.V(2).Infof("NodePublishVolume: volume(%s) mount on %s with service account token, VolumeContext: %v", volumeID, target, context)
```

Comment:
is it ok to log the `serviceAccountToken`? will it lead to security issue? I think print clientID, tenantID is ok

Reply:
Yes, I think we'd better don't print out token. I will print clientID and tenantID in the log.